packages/react-core/src/next/components/Dropdown/examples/DropdownBasic.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Dropdown, DropdownItem, DropdownList } from '@patternfly/react-core/next';
+import { Checkbox, Divider, MenuToggle } from '@patternfly/react-core';
+
+export const DropdownBasic: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [isPlain, setIsPlain] = React.useState(false);
+  const menuRef = React.useRef<HTMLDivElement>(null);
+
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = (_event: React.MouseEvent<Element, MouseEvent> | undefined, itemId: string | number | undefined) => {
+    // eslint-disable-next-line no-console
+    console.log('selected', itemId);
+    setIsOpen(false);
+  };
+
+  const toggleIsPlain = () => {
+    setIsPlain(!isPlain);
+  };
+
+  return (
+    <>
+      <div style={{ marginBottom: 20 }}>
+        <Checkbox
+          label="Plain menu"
+          isChecked={isPlain}
+          onChange={toggleIsPlain}
+          aria-label="plain menu checkbox"
```

Comment:
The `aria-label` can be removed since the label should apply an accessible name for the checkbox.